packages/engine/paima-rest/src/controllers/BasicControllers.ts

```diff
@@ -397,3 +398,153 @@ export class TransactionContentController extends Controller {
     }
   }
 }
+
+type GetLogsResponse = Result<
+  {
+    topic: string;
+    address: string;
+    blockNumber: number;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    data: { [fieldName: string]: any };
+    tx: number;
+    idx: number;
```

Comment:
It's probably not clear from the name that this is meant to represent the order in which this log was executed within the transaction

Reply:
I renamed it to `logIndex`. Only caveat is that if we wrap this to implement `eth_getLogs`, we need to remember that this is the index of the log in the tx, and the other one is the index of the log in the block. I guess we may be able to compute the index of the log out of the tx index and the log index though?

Reply:
Or maybe we may want to change this field to be the log index in the block to be consistent with that? I just used the index in the tx because it's what was described in https://github.com/PaimaStudios/paima-engine/issues/373

Reply:
ah good catch. I feel it should probably by the index in the block instead of in the transaction then